backend/src/api/album.rs

```diff
@@ -167,3 +169,87 @@ pub fn insert_share_token(rows: InsertShareToken, conn: &Connection) -> Result<(
 
     Ok(())
 }
+
+#[derive(Clone, Default, Deserialize)]
```

Comment:
Probably still want camelCase attribute on these so we dont forget for future changes?

Reply:
It currently has to be camelCase and snake_case so neither is correct, it only works because it has single words. I think it's fine because I don't know what else we would add but I'll add a comment to it.